src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1029,6 +1073,35 @@ private Task<ICosmosDBAccount> CreateCosmosDbAsync()
                     .WithWriteReplication(Region.Create(configuration.RegionName))
                     .CreateAsync(cts.Token));
 
+        private async Task<IMySQLManagementClient> CreateAndProvisionMySQLServerAsync(string subnetId)
+        {
+            var mySQLManagementClient = new MySQLManagementClient(azureCredentials)
+            { SubscriptionId = configuration.SubscriptionId };
+
+            // Create Server.
```

Comment:
In case user is using existing VNET, we need to make sure MySqlSubnet has delegation on it, otherwise the server creation will fail. 
Pass **ISubnet subnet** as parameter instead of **string subnetId**

```
    if (!subnet.Inner.Delegations.Any())
    {
        subnet.Parent.Update().UpdateSubnet(subnet.Name).WithDelegation("Microsoft.DBforMySQL/flexibleServers");
        await subnet.Parent.Update().ApplyAsync();
    }
```

Reply:
fixed in latest commit